### ISSUE TITLE
🤯 We are reinventing the wheel: WP sitemaps

### ISSUE THREAD

**Issue by szepeviktor** [leonstafford/WordPressURLDetector#7]
It seems like we write code that has already been written 😢 
https://github.com/WordPress/wordpress-develop/tree/master/src/wp-includes/sitemaps

Should we use/extend WordPress sitemaps classes?
What do you think?

**Comment by szepeviktor** [comment]
It is very easy to parse sitemaps and retrieve URL-s from them.

**Comment by leonstafford** [comment]
Definitely! Have rm’d current implementation already, as felt too complex. @john-shaffer has nicer code in leonstafford/wp2static-addon-advanced-crawling and I’ll checkout this WP stuff, thanks!
